core/src/main/java/com/google/errorprone/bugpatterns/UnusedVariable.java

```diff
@@ -149,6 +149,8 @@ public final class UnusedVariable extends BugChecker implements CompilationUnitT
   // TODO(ghm): Find a sensible place to dedupe this with UnnecessarilyVisible.
   private static final ImmutableSet<String> ANNOTATIONS_INDICATING_PARAMETERS_SHOULD_BE_CHECKED =
       ImmutableSet.of(
+          "com.google.errorprone.refaster.annotation.AfterTemplate",
+          "com.google.errorprone.refaster.annotation.BeforeTemplate",
```

Comment:
Of this list, currently there's only a test for `javax.inject.Inject`, so I didn't add any test cases for these new annotations, either. Let me know if I should. (I did test these changes against a collection of real Refaster templates.)